README.md

```diff
@@ -1,5 +1,22 @@
-# Monitor 404s
+# govuk-cdn-logs-monitor
```

Comment:
Might be useful to keep this in line with the styleguide.

https://github.com/alphagov/styleguides/blob/master/use-of-READMEs.md